header/p2p/server.go

```diff
@@ -52,19 +54,27 @@ func (serv *ExchangeServer) Stop(context.Context) error {
 
 // requestHandler handles inbound ExtendedHeaderRequests.
 func (serv *ExchangeServer) requestHandler(stream network.Stream) {
+	err := stream.SetReadDeadline(time.Now().Add(readDeadline))
```

Comment:
This only set's a read deadline on the host's end right?

Reply:
yes

Reply:
ack